modules/service/deployment/0.1/facets.yaml

```
@@ -317,11 +317,8 @@ spec:
             readiness_exec_command:
               type: array
               title: Readiness Exec Command
-              pattern: '^([^,]+)(,\s*[^,]+)*$'
-              x-ui-placeholder: "Enter readiness exec for the Pod, commands are separated
-                by commas"
-              x-ui-error-message: "Value doesn't match pattern, commands needs to
-                be comma separated"
+              x-ui-placeholder:  "Add each command argument as a new entry"
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)